app/indexers/sorting_indexer.rb

```diff
@@ -7,8 +7,7 @@ def initialize(resource:)
 
   def to_solr
     {
-      title_ssort: sort_title,
-      updated_at_dtsi: resource.updated_at
```

Comment:
what is this bit about?

Reply:
@hackmastera Valkyrie does `updated_at_dtsi` now.